configuration/config2/ConfigItemValueSet.cpp

```diff
@@ -31,7 +31,7 @@ void ConfigItemValueSet::addCfgValue(const std::shared_ptr<CfgValue> pNewValue)
     else 
     {
         std::string msg = "A value already exists for " + pNewValue->getName();
```

Comment:
Consider providing more context: 
eg "ValueSet: "this->getName()" + "Already contains a value named: " + pNewValue->getName()